user/configure-device.rst

```diff
@@ -76,7 +80,8 @@ Once it's installed, we need to configure it to connect to our OpenWISP2
 controller. To do that, edit the config file located at
 ``/etc/config/openwisp``.
 
-You will see the default config file, something like the following:
+You will see the default config file, something like the following: 
+(If your instance lacks some of the lines in the following, please add them)
```

Comment:
same here, no capital `I` in `if` and move colon to after the closing parentheses, eg:
```
following (if your instance lacks some of the lines in the following, please add them):
```